src/test/scala/io/iohk/ethereum/vm/OpCodeGasSpecPostEip2929.scala

```diff
@@ -203,13 +210,13 @@ class OpCodeGasSpecPostEip2929 extends AnyFunSuite with OpCodeTesting with Match
     val table = Table[UInt256, UInt256, Boolean, BigInt, BigInt](
       ("offset", "value", "alreadyAccessed", "startGas", "expectedGasConsumption"),
       (0, 1, true, G_callstipend + 1, G_sload),
-      (0, 1, false, G_sload + G_cold_account_access, G_sload + G_cold_account_access),
+      (0, 1, false, G_callstipend + 1, G_sload + G_cold_sload),
```

Comment:
I was wondering why the '+1' here, but the yellow paper answered me: the call will fail if there is less than or EQUALS to G_callstipend gas left.

Reply:
I wonder if we should start adding such tidbits of knowledge into the codebase.
//YP, page X, "quote"